source/releases/stable/1.4.2.rst

```diff
@@ -6,12 +6,12 @@ Synfig 1.4.2 Release Notes
 
 Synfig Studio 1.4.2 delivers some improvements and critical bugfixes. Please read below for details.
 
-Improved Lottie Exporter plugin
-----------------------------------------
+Improved export for Web
+------------------------
 
-* Added support for exporting Advanced Outline layer (`#2199 <https://github.com/synfig/synfig/pull/2199>`_).
-* Added support for exporting Blur layers (`#1505 <https://github.com/synfig/synfig/pull/1505>`_).
-* Added support for exporting Outline layer without variable widths (`#1647 <https://github.com/synfig/synfig/pull/1647>`_).
```

Comment:
I don't think it's not available: I believe it wasn't a feature export "without variable width", it was a constraint, that was solved by "Added support for exporting Advanced Outline layer (`#2199 <https://github.com/synfig/synfig/pull/2199>`_)."

Reply:
No, this is a different feature, related with ignoring any line width, for any outline layer - https://github.com/synfig/synfig/pull/1647
This feature was requested by me in this issue - https://github.com/synfig/synfig/issues/1217

Currently the code has this feature included, but it is not possible to enable this mode from UI - it is available in [extended plugin system](https://github.com/synfig/synfig/pull/1647#issuecomment-679846675), which is not in 1.4.x branch yet. ^__^